src/app/services/JoinerService.ts

```diff
@@ -12,12 +12,16 @@ import { MGPValidation } from '../utils/MGPValidation';
     providedIn: 'root',
 })
 export class JoinerService {
+
     public static VERBOSE: boolean = false;
 
     private observedJoinerId: string;
 
     private joinerUnsubscribe: MGPOptional<Unsubscribe> = MGPOptional.empty();
 
+    public static readonly USER_ALREADY_IN_GAME: () => string = () => $localize`You cannot join this game because you are already in another one.`;
```

Comment:
Juste pour double checker: est-ce que ça s'affiche si tu rejoins deux fois le même jeu ? Si oui, faut adapter le message (c'est pas "another one"). Si non, feel free to résoudre

Reply:
Dans une certaine mesure, on compte authoriser dans le futur le multi-tab ou simplement de rejoindre sa partie après l'avoir quitté donc l'erreur ne devrais pas s'afficher du tout.
Hors, là c'est le cas donc changer ce message n'est pas utile ni souhaitable.
Du coup j'ai ticketté cet issue.